libusb/os/windows_common.c

```diff
@@ -888,7 +894,7 @@ const struct usbi_os_backend usbi_backend = {
 	windows_init,
 	windows_exit,
 	windows_set_option,
-	windows_get_device_list,
+	NULL, /* get_device_list */
```

Comment:
Are you sure this is optional? Looking at `libusb_get_device_list` implementation in `core.c`, it looks like the function pointer is used without a check.

Reply:
I believe this is correct. It is how the backend communicates to core that it has hotplug support. The LIBUSB_CAP_HAS_HOTPLUG is set if the backend does not have a get_device_list function pointer. And if LIBUSB_CAP_HAS_HOTPLUG is set, the backend function is not called.

Reply:
Thanks for your comment @fabiensanglard . I understood things in the same way than @tormodvolden and implemented accordingly. This function pointer is also set to NULL on Linux for instance.